tests/test-verify-volume-mount.yml

```
@@ -55,6 +55,8 @@
     msg: >-
       Found unexpected mount state for volume
       '{{ storage_test_volume.name }}' device
+      (expected matches '{{ storage_test_mount_expected_match_count|int }}';
+      found matches '{{ storage_test_mount_device_matches|length }}')
```

```suggestion
      found matches '{{ storage_test_mount_device_matches | length }}')
```
ansible-lint likes spaces around the ` | ` expressions

---

Fixed.